operators/src/source/gdal_source/mod.rs

```diff
@@ -115,7 +114,7 @@ pub enum TimeReference {
 pub struct GdalDatasetParameters {
     pub file_path: PathBuf,
     pub rasterband_channel: usize,
-    pub geo_transform: GdalDatasetGeoTransform,
+    pub geo_transform: GdalDatasetGeoTransform, // TODO: discuss if we need this at all
```

Comment:
Maybe to be able to read raster files (images) without geotransform? 
Or: to avoid reading the metadata from disk every time. But maybe gdal reads it anyway.

Reply:
i do not know if gdal supports data without a geotransform. It guess it will report a pixel size of 1,-1. How can we decide what is right? Or will the json overrule the dataset? I guess this is part of the discussion what a dataset def really is.